src/main/java/com/findmyrecycling/fmrenterprise/dao/IFacilityRetrofitDAO.java

```diff
@@ -10,7 +10,7 @@
 public interface IFacilityRetrofitDAO {
 
     //TODO update method when we get an actual endpoint
-    @GET("/FMRJSONdata.md")
+    @GET("/https://github.com/Smoofington/FindMyRecyclingIntellijIDEA/blob/master/FMRJSONdata.md")
```

Comment:
The base URL should be (and may be) in the RetrofitClientInstance.  If so, it should not be here.  Also, this URL should not start with /, if it is an absolute URL.